code/modules/multiz/structures.dm

```diff
@@ -22,6 +22,11 @@
 	var/climb_time = 2 SECONDS
 	var/static/list/climbsounds = list('sound/effects/ladder.ogg','sound/effects/ladder2.ogg','sound/effects/ladder3.ogg','sound/effects/ladder4.ogg')
 
+	var/static/radial_ladder_down = image(icon = 'icons/screen/radial.dmi', icon_state = "radial_ladder_down")
```

Comment:
This doesn't really seem necessary, you could just 
````
#define RADIAL_LADDER_OPTIONS list("up" =  image(icon = 'icons/screen/radial.dmi', icon_state = "radial_ladder_up"), "down" = image(icon = 'icons/screen/radial.dmi', icon_state = "radial_ladder_down"))

var/direction = show_radial_menu(M, src,  RADIAL_LADDER_OPTIONS, require_near = !(isEye(M) || isobserver(M)))
````
Images and lists aren't expensive enough to bother with precaching them like this imo.